modules/fio-uninstall-console.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * security/file_integrity_operator/fio-uninstalling.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="fio-uninstall-console_{context}"]
+= Uninstall the File Integrity Operator using the web console
+
+To remove the File Integrity Operator, you must first delete the `FileIntegrity` objects in the FIO namespace. After the objects are removed, you can then remove the Operator and its namespace.
+
+.Prerequisites
+
+* Access to an {product-title} cluster that uses an account with `cluster-admin` permissions.
+* The File Integrity Operator is installed.
+
+.Procedure
+
+To remove the File Integrity Operator by using the {product-title} web console:
```

Comment:
I believe we are going away from these introductory statements before a procedure, but I can't find anything definitive.